istio-control/istio-config/templates/configmap-envoy.yaml

```diff
@@ -74,6 +74,54 @@ data:
                     route:
                       cluster: in.9901
                       timeout: 0.000s
+        {{- if .Values.global.sds.enabled }}
```

Comment:
Pilot has support for K8S signed certificates - which are needed for inject, etc.
And the feedback in ToC was to remove the sidecar if I remember correctly.